docs/operator-manual/argocd-cm.yaml

```
@@ -205,6 +205,12 @@ data:
       generate:
         command: [kasane, show]
 
+  # A set of settings that allow enabling or disabling the config management tool
```

```suggestion
  # A set of settings that allow enabling or disabling the config management tool.
  # If unset, each defaults to "true".
```

Just to be super clear. :-)

---

Sounds good! Accepted